pkg/test/env/istio.go

```diff
@@ -95,9 +95,6 @@ var (
 	// ChartsDir is the Kubernetes Helm chart directory in the repository
 	ChartsDir = path.Join(IstioSrc, "install/kubernetes/helm")
```

Comment:
Note that `install/kubernetes/helm` is also not valid path from `1.6`. `ChartsDir` is referenced in [ServiceAccountFilePath](https://github.com/istio/istio/pull/24174/files#diff-b7bc2684a03df2e326b38e1c6d0a7624R105) which also did not exists.

Reply:
I noticed this previously. Looks like the only test using it is skipped so it doesn't fail. I recommend we just wait for 1.7 when mixer is gone then we can just delete it all?

Reply:
Sounds good, I think deleting `IstioChartDir` is ok with this PR as it is not referenced anywhere in the code.

Reply:
Not sure if it is safe to remove `IstioChartDir`.
Leave this to @howardjohn